devdox/app/routes/api_keys_route.py

```
@@ -0,0 +1,36 @@
+        PostApiKeyService, Depends(PostApiKeyService.with_dependency)
+    ],
+) -> JSONResponse:
+    """
+    Generate a new API key
+    """
+    db_id, plain_key = await service.generate_api_key(
+        user_claims=user_claims
+    )
+
+    return APIResponse.success(
+        message=constants.API_KEY_GENERATED_SUCCESSFULLY, data={ "id": db_id, "api_key": plain_key }
```

Flake8 (7.2.0)

[error] whitespace after '{'

(E201)